lib/core/connector_settings.rb

```diff
@@ -80,20 +83,33 @@ def scheduling_settings
       self[:scheduling]
     end
 
+    def filtering
+      Utility::Commons.return_if_present(@elasticsearch_response[:filtering], DEFAULT_FILTERING)
```

Comment:
What is `:filtering`?

Reply:
See the [implementation document](https://docs.google.com/document/d/1llPDGzduj8lST0N45v_G4PB0s0RUOBFurOIsBAD-Fyg/edit?pli=1#heading=h.l2heyce8rz9r)